templates/todo/common/infra/bicep/app/apim-api.bicep

```diff
@@ -0,0 +1,98 @@
+param name string
+
+@description('The name of the API')
+@minLength(1)
+param apiName string
+
+@description('The Display Name of the API')
+@minLength(1)
+param apiDisplayName string
+
+@description('The description of the API')
+@minLength(1)
+param apiDescription string
+
+@description('The path of the API')
+@minLength(1)
+param apiPath string
+
+@description('URL for the backend API')
+param apiBackendUrl string
+
+resource apimService 'Microsoft.ApiManagement/service@2021-08-01' existing = {
```

Comment:
I like to do existing refs at the bottom of files

Reply:
fixed